tools/topology/topology2/include/components/google-rtc-aec.conf

```diff
@@ -47,6 +47,15 @@ Class.Widget."google-rtc-aec" {
 		token_ref	"comp.word"
 	}
 
+	Object.Control.bytes."1" {
+		access [
+			tlv_write
+			tlv_callback
```

Comment:
is callback a kernel or userspace permission?

Reply:
This is a cherry pick from main.  And "access" in this context refers (I think!) to the API variant used to access the control's data.  "callback" is what needs to be used for large TLV blocks because they can't be sent in a single IPC command.